website/app/styles/spacing/index.scss

```
@@ -120,3 +120,8 @@ pre[class*="language-"].doc-markdown-pre.doc-code-block__code-snippet + pre[clas
     margin-top: 32px;
   }
 }
+
+.doc-layout + .doc-markdown-p,
+.doc-badge + .doc-markdown-p {
+  margin-top: 16px;
+}
```

with this syntax, adding new use cases in the future (if needed) should be much easier

```suggestion

// DOC components without a margin bottom followed by a markdown paragraph

.doc-badge,
.doc-layout {
  + .doc-markdown-p {
    margin-top: 16px;
  }
}
```